NetworkPkg/Ip6Dxe/Ip6Nd.c

```diff
@@ -1281,15 +1302,29 @@ Ip6SendNeighborAdvertise (
   }
 
   Target = (EFI_IPv6_ADDRESS *)NetbufAllocSpace (Packet, sizeof (EFI_IPv6_ADDRESS), FALSE);
-  ASSERT (Target != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (Target == NULL) {

```

Comment:
Looks like `NetBufFree (IcmpHead);` should be in this block.

Reply:
Same in block below in same function.

Reply:
These `NetbufAllocSpace()` calls appear to be carving out space from an already allocated buffer so I'll drop my comment on them.